hedera-node/src/test/java/com/hedera/services/ledger/TransactionalLedgerTest.java

```diff
@@ -548,6 +562,122 @@ void validationFailsAsExpected() {
 		assertEquals(ACCOUNT_STILL_OWNS_NFTS, subject.validate(4L, scopedCheck));
 	}
 
+	@Test
+	void validationFailsWithMissingHbarAllowance() {
+		given(backingAccounts.getRef(1L)).willReturn(account1);
+		given(backingAccounts.getImmutableRef(1L)).willReturn(account1);
+		given(backingAccounts.contains(1L)).willReturn(true);
+
+		subject.begin();
+		subject.set(1L, LONG, 123L);
+		subject.set(1L, FLAG, false);
+		subject.set(1L, OBJ, "DEFAULT");
+		subject.set(1L, HBAR_ALLOWANCES, MISSING);
+		subject.commit();
+
+		assertEquals(SPENDER_DOES_NOT_HAVE_ALLOWANCE, subject.validate(1L, scopedCheck));
+	}
+
+	@Test
+	void validationFailsWithInsufficientHbarAllowance() {
+		given(backingAccounts.getRef(1L)).willReturn(account1);
+		given(backingAccounts.getImmutableRef(1L)).willReturn(account1);
+		given(backingAccounts.contains(1L)).willReturn(true);
+
+		subject.begin();
+		subject.set(1L, LONG, 123L);
+		subject.set(1L, FLAG, false);
+		subject.set(1L, OBJ, "DEFAULT");
+		subject.set(1L, HBAR_ALLOWANCES, INSUFFICIENT);
+		subject.commit();
+
+		assertEquals(AMOUNT_EXCEEDS_ALLOWANCE, subject.validate(1L, scopedCheck));
+	}
+
+//	@Test
```

Comment:
Were these commented-out tests still important?

Reply:
Not relevant, deleting them.